app/models/user.rb

```diff
@@ -13,10 +13,10 @@ class User < ApplicationRecord
   enum role: { provider: 'provider', renter: 'renter' }
 
   validates :first_name, :last_name, presence: true
-  validates :prefecture,
-            inclusion: { in: %w[Aichi Akita Aomori Chiba Ehime Fukui Fukuoka Fukushima Gifu Gunma Hiroshima
-                                Hokkaido Hyogo Ibaraki Ishikawa Iwate Kagawa Kagoshima Kanagawa
-                                Kochi Kumamoto Kyoto Mie Miyagi Miyazaki Nagano Nagasaki Nara Niigata
-                                Okayama Okinawa Oita Osaka Saga Saitama Shiga Shimane Shizuoka Tochigi
-                                Tokushima Tokyo Tottori Toyama Wakayama Yamagata Yamaguchi Yamanashi], allow_nil: false }
+  # validates :prefecture,
+  #           inclusion: { in: %w[Aichi Akita Aomori Chiba Ehime Fukui Fukuoka Fukushima Gifu Gunma Hiroshima
+  #                               Hokkaido Hyogo Ibaraki Ishikawa Iwate Kagawa Kagoshima Kanagawa
+  #                               Kochi Kumamoto Kyoto Mie Miyagi Miyazaki Nagano Nagasaki Nara Niigata
+  #                               Okayama Okinawa Oita Osaka Saga Saitama Shiga Shimane Shizuoka Tochigi
+  #                               Tokushima Tokyo Tottori Toyama Wakayama Yamagata Yamaguchi Yamanashi], allow_nil: false }
 end
```

Comment:
Wait why are we commenting these out again? I may have missed the details during conversation, sorry.